Edited grammar and presentation (#37105)
freeCodeCamp/freeCodeCamp: README.md

freeCodeCamp.org offers several free developer certifications. Each of these certifications involves building 5 required web app projects, along with hundreds of optional coding challenges to help you prepare for those projects. We estimate that each certification will take a beginning programmer around 300 hours to earn.

> Each of these 30 projects in the freeCodeCamp.org curriculum have their own agile user stories and automated tests. These help you build up your project incrementally and ensure you've fulfilled all the user stories before you submit it.

Reviewer: Please revert.
```suggestion
Each of these 30 projects in the freeCodeCamp.org curriculum has its own agile user stories and automated tests. These help you build up your project incrementally and ensure you've fulfilled all the user stories before you submit it.
```

You can pull in these test suites through [freeCodeCamp's CDN](https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js). This means you can build these projects on websites like CodePen and Glitch - or even on your local computer's development environment.

> Once you’ve earned a certification, you will always have it. You will always be able to link to it from your LinkedIn or Résumé. And when your prospective employers or freelance clients click that link, they’ll see a verified certification specific to you.

Reviewer: Do not capitalize resume.  Also, it should not have accent marks.
